qiskit_aer/quantum_info/states/aer_densitymatrix.py

```diff
@@ -0,0 +1,354 @@
+            dims (int or tuple or list): Optional. The subsystem dimension of
+                                         the state (See additional information).
+            configs (kwargs): configurations of :class:`AerDensityMatrix`. `_aer_state` and `method`
+            are valid.
+        Raises:
+            AerError: if input data is not valid.
+        Additional Information:
+            The ``dims`` kwarg is used to ``AerDensityMatrix`` constructor.
+        """
+        if '_aer_state' in configs:
+            self._aer_state = configs.pop('_aer_state')
+        else:
+            if 'method' not in configs:
+                configs['method'] = 'density_matrix'
+            elif configs['method'] != 'density_matrix':
+                method = configs['method']
+                raise AerError(f'Method {method} is not supported')
+            if isinstance(data, (QuantumCircuit, Instruction)):
+                data, aer_state = AerDensityMatrix._from_instruction(data, None, configs)
+            elif isinstance(data, list):
+                data = self._from_1d_array(np.array(data, dtype=complex))
+                data, aer_state = AerDensityMatrix._from_ndarray(data, configs)
+            elif isinstance(data, np.ndarray):
+                data = self._from_1d_array(data)
+                data, aer_state = AerDensityMatrix._from_ndarray(data, configs)
+            elif isinstance(data, AerDensityMatrix):
+                aer_state = data._aer_state
+                if dims is None:
+                    dims = data._op_shape._dims_l
+                data = data._data.copy()
+            elif isinstance(data, DensityMatrix):
+                data, aer_state = AerDensityMatrix._from_ndarray(np.array(data.data,
+                                                                          dtype=complex), configs)
+            elif hasattr(data, 'to_operator'):
+                # If the data object has a 'to_operator' attribute this is given
+                # higher preference than the 'to_matrix' method for initializing
+                # an Operator object.
+                op = data.to_operator()
+                data, aer_state = AerDensityMatrix._from_ndarray(op.data, configs)
+                if dims is None:
+                    dims = op.output_dims()
+            elif hasattr(data, 'to_matrix'):
+                # If no 'to_operator' attribute exists we next look for a
+                # 'to_matrix' attribute to a matrix that will be cast into
+                # a complex numpy matrix.
+                data, aer_state = AerDensityMatrix._from_ndarray(
+                    np.asarray(data.to_matrix(), dtype=complex), configs)
+            else:
+                raise AerError(f'Input data is not supported: type={data.__class__}, data={data}')
+
+            self._aer_state = aer_state
+
+        super().__init__(data, dims=dims)
+
+        self._result = None
+        self._configs = configs
+
```

Comment:
```suggestion

    def seed(self, value=None):
        """Set the seed for the quantum state RNG."""
        if value is None or isinstance(value, int):
            self._aer_state.set_seed(value)
        else:
            raise AerError(f'This seed is not supported: type={value.__class__}, value={value}')

```
seed method is missing.

Reply:
I had overlooked this. So Terra's QuantumState.seed is called and the effect of the call to seed() in test_aer_densitymatrix.py is inadequate. OK, I will fix this as you suggested.